okhttp/src/main/java/okhttp3/internal/concurrent/Task.kt

```diff
@@ -48,7 +48,7 @@ package okhttp3.internal.concurrent
  */
 abstract class Task(
   val name: String,
-  val cancelable: Boolean = true
```

Comment:
I moved the default value to TaskQueue.schedule(). (I don’t like repeating the defaults; I find that hard to reason about.)